prime-router/settings/organizations.yml

```
@@ -104,13 +104,13 @@
       organizationName: ignore
       topic: full-elr
       customerStatus: active
-      #      schemaName: metadata/fhir_transforms/fhir-transform-sample
+      schemaName: metadata/fhir_transforms/default-sender-transform
       format: HL7
     - name: ignore-fhir      # Use this to test fhir
       organizationName: ignore
       topic: full-elr
       customerStatus: active
-      #      schemaName: metadata/fhir_transforms/fhir-transform-sample
+      schemaName: metadata/fhir_transforms/fhir-transform-sample
```

I think we probably want this to use the new default too.

```suggestion
      schemaName: metadata/fhir_transforms/default-sender-transform
```